### ISSUE TITLE
% Complete does not update on my mini 12864 LCD Display

### ISSUE THREAD

**Issue by BNeuf73** [FYSETC/FYSETC-SPIDER#47]
Am I missing something in my printer.cfg that my % complete does not update on the LCD screen. 
It does update in Octoprint just fine.

All other functions work correctly on screen. Using Spider V1.0 board with my Voron 2.4 setup.

**Comment by GerogeFu** [comment]
This should be related to klipper firmware, better to update your klipper, then have a try.

**Comment by yojoebosolo** [comment]
did you ever get this sorted? I have the same issue.

**Comment by BNeuf73** [comment]
Yes I did get it resolved. The gcode command M73 will resolve the problem. You can either enter this manually in Octoprint command line before you start printing or add it to the "start gcode" in your printer settings in cura.
